codegen/projections/cbor_event_streams/.yardopts

```diff
@@ -0,0 +1,2 @@
+--title "RpcV2Cbor Event Streams Test Service"
+--hide-api private
```

Comment:
We can't put it into the existing cbor service because that service is defined by Smithy and we can't add new operations to it (that I've been able to find).

We DO support http23 and http1 in the same client - in fact this example service has that (see the non-streaming operation).